src/main/java/com/pileproject/drivecommand/command/CommandBase.java

```diff
@@ -32,27 +35,28 @@ public CommandBase(CommandType type, Map<String, Object> args) {
     }
 
     /**
-     * Get its command type
+     * Get this command type.
```

Comment:
Maybe we need some agreement on this and other developing conventions.
These should be shared over our team and noted on a wiki system or a website.